src/Magento/FunctionalTestingFramework/Util/ModuleResolver.php

```diff
@@ -265,7 +270,8 @@ private function globRelevantPaths($testPath, $pattern)
         }
 
         foreach ($relevantPaths as $codePath) {
-            $mainModName = basename(str_replace($pattern, '', $codePath));
+            $allComponents = $this->getRegisteredModuleList();
```

Comment:
Need to do two things here:
* Strip `/Test/Mftf` off end of codepath
* Assign to $modulePaths according to $allComponents value match (Magento_Store -> Store) to match module list